### ISSUE TITLE
[Test] Upscale tests!

### ISSUE THREAD

**Issue by toyxyz** [s9roll7/animatediff-cli-prompt-travel#21]
I tried a few ways to upscale the video generated by AnimateDiff.

Webui ControlNet(tile+lineart+TemproalNet) batch img2img

Flickering is visible throughout the video, just like a normal batch img2img.

https://github.com/s9roll7/animatediff-cli-prompt-travel/assets/8006000/8c49544c-5219-4754-a918-090a01dd8cec

AnimateDiff ControlNet tile

Much cleaner than TemporalNet! However, due to high VRAM consumption, upscaling beyond 1024x1536 is not possible.

https://github.com/s9roll7/animatediff-cli-prompt-travel/assets/8006000/e333a239-b1c2-45d1-8366-b9aa7ece685b

AnimateDiff ControlNet tile - > Webui Adetailer + NMKD YandereNeo (4x)

I upscaled the video to 1024x1536 with AnimateDiff, then used Adetailer to enhance the detail on the character's face, then upscaled it to 4K resolution with NMKD YandereNeo (4x). It became very sharp!

https://github.com/s9roll7/animatediff-cli-prompt-travel/assets/8006000/ba7b2da0-505a-413f-b558-df62a4c7ec10

**Comment by xnmt9** [comment]
nice results! what did you feed in as tile input, all of the frames from the smaller video?

**Comment by toyxyz** [comment]
> nice results! what did you feed in as tile input, all of the frames from the smaller video?

Yes. I placed all the 512x768 images back into the ControlNet tile folder.

![image](https://github.com/s9roll7/animatediff-cli-prompt-travel/assets/8006000/ed686cb9-b1e7-48b1-9a87-80ba0da3a3c2)

**Comment by xnmt9** [comment]
cool 👍🏽 I got it to work but had to set upscale strength pretty low, otherwise a lot of jitteriness

**Comment by PattAtk** [comment]
> > nice results! what did you feed in as tile input, all of the frames from the smaller video?
> 
> Yes. I placed all the 512x768 images back into the ControlNet tile folder.
> 
> ![image](https://user-images.githubusercontent.com/8006000/264287146-ed686cb9-b1e7-48b1-9a87-80ba0da3a3c2.png)

Thanks for all your work! I'm having trouble getting Animatediff upscale to recognize Controlnet Tile input images. I put the small base images in the folder as you suggested (the same controlnet images folder as the base generation), and even tried setting "use_preprocessor":true, under controlnet_tile in upscale_config but the result looks the same with or without the small frames in the tile folder. Is there any indication in the command line process that indicates controlnet tile was using the input images? Because I only saw this and no sign of preprocessor being used. Nothing in upscale output folder either.

12:33:35 INFO     use_controlnet_tile=True                                                               generate.py:886
         INFO     use_controlnet_line_anime=False                                                        generate.py:887
         INFO     use_controlnet_ip2p=True                                                               generate.py:888
         INFO     controlnet_conditioning_scale=[1.0, 0.5]                                               generate.py:890
         INFO     guess_mode=[False, False]                                                              generate.py:891
         INFO     control_guidance_start=[0.0, 0.0]                                                      generate.py:892
         INFO     control_guidance_end=[1.0, 1.0]                                                        generate.py:893
Upscaling... 100% ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 20/20  [ 0:00:52 < 0:00:00 , 0 it/s ]

The only thing different was my base frames are -W 256 -H 384 -L 18 -C 16 and my upscale is to -W 512

My upscale settings:

  "upscale_config": {
    "scheduler": "ddim",
    "steps": 20,
    "strength": 0.5,
    "guidance_scale": 10,
    "controlnet_tile": {
      "enable": true,
      "use_preprocessor":true,
      "preprocessor":{
        "type" : "tile_resample",
        "param":{
          "down_sampling_rate":2.0
        }
      },
      "controlnet_conditioning_scale": 1.0,
      "guess_mode": false,
      "control_guidance_start": 0.0,
      "control_guidance_end": 1.0
    },
